content/asset-tracking/using-the-sdks.textile

```diff
@@ -600,8 +601,82 @@ subscriber.nextLocationUpdateIntervals
+
+The location animator then recalculates each step duration to evenly take up the animation duration, so the overall animation of the trackable is smooth.
+
+Each step is animated at 60 FPS, and this applies to both the map marker and the camera position.
+
+More information can also be found in the location animator "reference docs":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/ui-sdk/ui-sdk/com.ably.tracking.ui.animation/-location-animator/index.html.
+
+h3(#location-animator-create-instance). Create an instance
+
+Creating an instance of the location animator:
+
+```[kotlin]
+val locationAnimator: LocationAnimator = CoreLocationAnimator(
+  intentionalAnimationDelayInMilliseconds,
+  animationStepsBetweenCameraUpdates
+) 
+```
+
+There are two optional parameters:
+
+- intentionalAnimationDelayInMilliseconds := The higher the delay, the more buffer there is to allow for unexpected delays of location updates from the publisher. Also moves the actual position of the trackable backwards in time. The higher the setting the more network issues and delays are allowed for, but the position updates will be less realtime in nature. By default this is set to two seconds, but you should test what value works best for your use case.
+- animationStepsBetweenCameraUpdates := By default, when a step is being animation, the camera position is also updated. But this can be changed so that the camera position updates, for example, on every five location updates.  
+
+h3(#location-animator-stop). Stop the location animator
+
+As with other components, you can stop the location animator with the @stop@ method:
+
+```[kotlin]
+locationAnimator.stop()
+```
+
+The location animator should be stopped when no longer used to conserve resources.
+
+h3(#location-animator-observing-positions). Observing the animated positions
+
+The location animator exposes a positions flow:
+
+```[kotlin]
+locationAnimator.positionsFlow
+  .onEach { trackablePosition -> }
+  .launchIn(scope)
+```
+
+This provides data you can use to move your map marker on the map, or when you want to move your trackable in the screen.
+
+A camera positions flow is also provided:
+
+```[kotlin]
+locationAnimator.cameraPositionsFlow
+  .onEach { cameraPosition -> }
+  .launchIn(scope)
+```
+
+This provides data you can use to move your camera whenever a new camera position is read. 
+
+h3(#location-animator-location-updates-animate). Providing location updates to animate
+
+To use the animator you need to provide it with some information:
+
+```[kotlin]
+locationAnimator.animateLocationUpdate(locationUpdate, expectedIntervalBetweenLocationUpdatesInMilliseconds)
+```
+
+- locationUpdate := The location update from the publisher.
```

Comment:
Location animator users only have access to the subscriber SDK, so perhaps we should not mention the publisher here? 🤔 
```suggestion
- locationUpdate := The location update from the subscriber.
```